bigquery_etl/glean_usage/templates/baseline_clients_first_seen_v1.init.sql

```diff
@@ -16,6 +16,9 @@ WITH
     SELECT
       client_info.client_id,
       sample_id,
+      -- Some Glean data from 2019 contains incorrect sample_id, so we
+      -- recalculate here; see bug 1707640
+      udf.safe_sample_id(client_info.client_id) AS sample_id,
```

Comment:
The tests will start to fail because they use a dummy value for the sample id.

Reply:
I'm going to make the `expect.yaml` match the actual output for now. It may be good to come back to this and fix up the tests to use full UUID values.

Reply:
That, or a single test row with a legitimate uuid value.